src/simularium/ISimulator.ts

```diff
@@ -46,4 +58,11 @@ export interface ISimulator {
     requestFrameByTime(time: number): void;
     /** request trajectory metadata */
     requestTrajectoryFileInfo(fileName: string): void;
+    /** request available metrics */
+    requestAvailableMetrics(): void;
+    /** request available plots */
+    requestPlotData(
+        data: Record<string, unknown>,
+        plots: Array<PlotConfig>
```

Comment:
can you describe what these args are?  it's hard to tell what `data` means in this context, better typing and/or naming would make it a bit clearer I hope